README.md

```diff
@@ -1,2 +1,127 @@
-# res_portfolio
-Basic portfolio website
+<a name="readme-top"></a>
+
+<div align="center">
+
+  <img src="./img/murple_logo.png" alt="microverse_logo" width="140"  height="auto" />
+  <br/>
+
+  <h3><b>Microverse README Template</b></h3>
+
+</div>
```

Comment:
- [ ] Dear @EmekaEmmanuel, Good job on the readme but Please update or remove this section, Readme should've updated according to the repo and the developer information.